index.html

```diff
@@ -66,6 +64,188 @@ <h2>I’am a Software Developer</h2>
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about-me">
+        <div class="about-text">
+          <h2>About me</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button class="about-button" type="button">Get my Resume</button>
+          .
+          <img class="side-image2" src="Icons/side_img2.png" alt="" />
+          <div class="text-line"></div>
+        </div>
+
+        <div class="tech-competencies">
+          <div class="language">
+            <ul>
+              <li>
+                <h3>
+                  <img src="Icons/listStyle/ls1.png" alt="List" />Language
+                </h3>
+              </li>
+              <li><p>Javascript</p></li>
+              <li><p>Ruby</p></li>
+              <li><p>Html</p></li>
+              <li><p>Css</p></li>
+            </ul>
+          </div>
+
+          <div class="framework">
+            <ul>
+              <li>
+                <h3>
+                  <img src="Icons/listStyle/ls2.png" alt="List" />Frameworks
+                </h3>
+              </li>
+              <li><p>Bootstrap</p></li>
+              <li><p>Ruby on Rails</p></li>
+              <li><p>RSPec</p></li>
+              <li><p>CapyBara</p></li>
+              <li><p>Selenium</p></li>
+            </ul>
+          </div>
```

Comment:
- [ ] Please the section has the margins (top, bottom, left, right) defined in the design. This is the [Link to Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) 

- [ ] PLEASE BELLOW IS THE ORIGINAL DESIGN
![gidi4](https://user-images.githubusercontent.com/63972670/159128224-0ffe4f37-a8a3-4b5b-ad04-51eedf3277a4.JPG)

- [ ]  BELLOW IS WHAT YOU DESIGNED


![WhatsApp Image 2022-03-19 at 4 18 05 PM (2)](https://user-images.githubusercontent.com/63972670/159128255-fb722332-1a46-4ffa-9d47-935665ca9e0e.jpeg)

Reply:
- [ ] As asked by the previous reviewer. Kindly implement it. The section doesn't have has the margins (top, bottom, left, right) defined in the design. Make sure to make Languages, Frameworks and Skills in the center.
- Your design:
<img width="433" alt="Screenshot 2022-03-19 at 11 24 35 PM" src="https://user-images.githubusercontent.com/70269507/159137417-bde93eb8-3cad-4264-9f53-d4bb605e9488.png">

- Figma design:
<img width="247" alt="Screenshot 2022-03-19 at 11 26 28 PM" src="https://user-images.githubusercontent.com/70269507/159137426-9e02ce11-d088-478e-ab99-bea1d434219b.png">